app.rb

```diff
@@ -122,4 +124,26 @@ def list_genres
   def add_music_album
     @music.add_music_album
   end
+
```

Comment:
- [ ] to start fixing the issue with the author not being saved I advise you to create a new instance of Author and pushing it to the author_list in the add_book method in the app class, similar to what you're doing with label, see the below example:
```
    author_obj= Author.new(author)
    @author_list.push(author_obj)
```